srsly/tests/test_json_api.py

```diff
@@ -204,3 +204,44 @@ def test_unsupported_type_error():
     f = numpy.float32()
     with pytest.raises(TypeError):
         s = json_dumps(f)
+
+
+def test_write_jsonl_gzip():
```

Comment:
Can you check the formatting on everything?

Reply:
Done (didn't apply `black` before, sorry).